api/envoy/config/trace/v3/dynamic_ot.proto

```diff
@@ -33,11 +33,11 @@ message DynamicOtConfig {
   string library = 1 [
     deprecated = true,
     (validate.rules).string = {min_len: 1},
-    (envoy.annotations.deprecated_at_minor_version) = "3.0"
+    (envoy.annotations.deprecated_at_minor_version) = "2.10"
```

Comment:
Is this an Envoy API version or an OT version? My understanding of this annotation is that it refers to an Envoy API version.

Reply:
i was just reading the relevant proto ...

afaict you are correct this refers to the proto version, altho given that there is no intention of increasing the api version im not sure that is what it _should_ be (separately its intended to refer to the version at which it becomes deprecated rather than the version at which it will be removed)

ill revert the version changes but i think the changelog change is correct